MarkBind/vue-strap
# Searchbar: Add agolia key

**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**

• [X] Enhancement to an existing feature

Part of [MarkBind/markbind#691](https://github.com/MarkBind/markbind/pull/691).

**What is the rationale for this request?**
We want the user to be able to easily use Agolia results with the `searchbar` component.

**What changes did you make? (Give an overview)**
Add the `agolia` key to the `searchbar` component. A user who wants the `searchbar` to use Agolia results can simply add the `agolia` key when defining the `searchbar`.

This works by adding the `agolia-search-input` id to the `input`, which will be used by the `agolia` initialisation code in Markbind's `page.ejs`.

**Provide some example code that this change will affect:**

```
<searchbar placeholder="Search" agolia menu-align-right></searchbar>
```

**Testing instructions:**
The component 
```
<searchbar placeholder="Search" agolia menu-align-right></searchbar>
```
should have the id `agolia-search-input` added to the `input` element.
